tests/integration/targets/keycloak_authz_custom_policy/policy/Containerfile.license

```
@@ -0,0 +1,4 @@
+// Copyright (c) Ansible Project
+// GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+// SPDX-License-Identifier: GPL-3.0-or-later
```

The comment signs aren't needed for `.license` files:
```suggestion
Copyright (c) Ansible Project
GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
SPDX-License-Identifier: GPL-3.0-or-later
```
Same for the other `.license` files.

---

Done for all license files.